_posts/2020-12-18-status-update.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "2020-12-18: IAC Team Status Update"
+author: sheenaajay
+categories:
+  - community
+  - releases
+  - dsc
+  - windows
+  - cloud-ci
+  - puppet-module-gem
+  - rubocop
+  - infrastructure
+  - team
+  - status
+tags:
+---
+
+## Christmas edition
+Puppet is getting ready for the Virtual Chirstmas Party.
+Santa, Food, Drinks, Sweets, Gaming, Charades and many more.
+
+The holiday season upon us, the IAC team would like to wish all of Puppets opensource community Happy Holidays and thank everyone for their contributions through the very challenging year of 2020.
+The IAC team will be taking a well-earned break to recharge the batteries.
```

Comment:
```suggestion
The IAC team will be taking a well earned break to recharge the batteries.
```

Reply:
Got error while adding commits on the same line. Pushed those changes now.